include/openssl/evp.h

```diff
@@ -531,11 +531,13 @@ int EVP_MD_block_size(const EVP_MD *md);
 unsigned long EVP_MD_flags(const EVP_MD *md);
 
 const EVP_MD *EVP_MD_CTX_md(const EVP_MD_CTX *ctx);
+# ifndef OPENSSL_NO_DEPRECATED_3_0
 int (*EVP_MD_CTX_update_fn(EVP_MD_CTX *ctx))(EVP_MD_CTX *ctx,
```

Comment:
The pattern used throughout evp.h seems to have this for the methods..
````
OSSL_DEPRECATEDIN_3_0 ....
````

Reply:
I don't know if this is the right/necessary thing to do
```
# ifndef OPENSSL_NO_CEPRECATED_3_0
OSSL_DEPRECATEDIN_3_0 int (*EVP ...
# endif
```

ping @levitte for info.

Reply:
That's the necessary thing to do.

Reply:
thanks; updated commit pushed.